app/models/katello/concerns/location_extensions.rb

```diff
@@ -8,6 +8,7 @@ module LocationExtensions
         before_create :set_katello_default
         before_save :refute_katello_default_changed
         before_destroy :deletable?
+        attr_accessible :katello_default
```

Comment:
Believe me I tried hard not to add this. Its there because 
```
Location.create!(:name => "Default Location") do |location|
  location.katello_default = true
end
```
Didnt work. I would get "cant mass assign" even though its technically not mass assigning there,